src/gazebo_plugins/ControlBoard.cc

```diff
@@ -109,41 +116,62 @@ GZ_REGISTER_MODEL_PLUGIN(GazeboYarpControlBoard)
 
         for (int n = 0; n < netList->size(); n++)
         {
-            yarp::os::ConstString driverName( netList->get(n).asString().c_str());
+            m_driverName = netList->get(n).asString();
```

Comment:
Ops, yes.  I'll fix it. 
I didn't want to add too much complexity but it turns out to have more facets then I though at the beginning.

To add more variability, are we allowed in gazebo to have 2 or more iCub models loaded at the same time?
I think it'll be bettere to use robot scoped name ad a unique key to register devices to avoid duplicated name, for example 2 let_arm controlboards coming from different robots. 
What do you think?